deviceComunicationDocumentation.MD

```diff
@@ -410,12 +379,11 @@ Obsah:
 Errory se dělí na dva druhy chyb - fatální a standartní. Normální chyby mají hodnotu 0.bytu
 nastavenou na 0 a fatální mají hodnotu 1.
 
-Chybový registr obsahuje informaci o tom jaká chyba nastala.
+Chybový registr obsahuje informaci, které konkrétní chyby nastaly.
```

Comment:
chyby - mnozne cislo.   Muze jich byt vice?

Reply:
Ano, například při initu může nastat více chyb zároveň (např. chyba senzoru a nízké napětí baterie). Je to sice velice nepravděpodobné, že by něco takového běžně nastalo, nicméně v rámci obecnosti píšu raději chyby.